src/App.js

```diff
@@ -1,26 +1,59 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, {useEffect, useState} from 'react';
+import Recipe from './Recipe';
 import './App.css';
 
-function App() {
+const App = () => {
+  const APP_ID = "0cc96450";
+  const APP_KEY = "14d029aed46efe535cde32ea66599608";
+
+  const [recipes, setRecipes] = useState([]); // recipes to display, initially set to empty array
+  const [search, setSearch] = useState(' '); // search state
+  const [query, setQuery] = useState('chicken'); // search terms we are querying by
+
+  useEffect( () => {
+    const getRecipes = async () => {
+      // can also use .then
+      const response = await fetch(`https://api.edamam.com/search?q=${query}&app_id=${APP_ID}&app_key=${APP_KEY}`); // external request
+      const data = await response.json(); // need to wait for response to come 
+      setRecipes(data.hits);
+      console.log(data.hits);
```

Comment:
nice job using console.log to debug! for future practice, it's generally good to clean up your commits by remove all your console.log statements once you're done using them to debut

Reply:
got it! i noticed that I forgot to delete the console.log line yesterday too while looking over the code with Ethan, but will do in the future :)